src/app/components/kudos-wisdom-wall-ideas/kudos-wisdom-wall-ideas.component.js

```diff
@@ -1,8 +1,10 @@
 (function () {
     var WisdomWallIdeasController = function ($scope, WisdomWallIdeasService){
+        $scope.showLoader = true;
```

Comment:
Perfect candidate to remove the `angular.controller` and just use the controller as a function.